app/policies/draw_policy.rb

```diff
@@ -46,6 +46,20 @@ def intent_actions?
     !user.student?
   end
 
+  def reminder?
+    record.pre_lottery? && !user.student?
+  end
+
+  def intent_reminder?
+    return false unless record.intent_deadline
+    reminder? && Time.zone.today <= record.intent_deadline
+  end
+
+  def locking_reminder?
+    return false unless record.locking_deadline
+    reminder? && Time.zone.today > record.intent_deadline
```

Comment:
Are we not setting a locking deadline as a Draw attribute? I feel like we should, and maybe validate that it is after the intent deadline (if present) so that this continues to work.